datasets/hebrew-sentiment/hebrew-sentiment.py

```diff
@@ -0,0 +1,123 @@
+"""HebrewSentiment: A Modern Hebrew Sentiment Analysis Dataset."""
```

Comment:
could you rename the paths using `hebrew_sentiment` ?
we're using snake case convention

Reply:
Done.